### ISSUE TITLE
Ditch ORM for SQLAlchemy Core

### ISSUE THREAD

**Issue by Antar1011** [Antar1011/Onix#48]
_(it's a little premature to put in this issue now, considering the [`backend` branch](https://github.com/Antar1011/Onix/tree/backend) isn't even in PR yet, much less merged, but since I'm pretty sure I'm going down the ORM route for now, I figure I should put this in before I lose the thread)_

As per SQLAlchemy's documentation, [ORM operations are slow](http://docs.sqlalchemy.org/en/latest/faq/performance.html#i-m-inserting-400-000-rows-with-the-orm-and-it-s-really-slow) compared to SQLAlchemy's Expression Language (or executing the SQL directly), so when it comes time to do performance tuning (0.9 milestone), this might be an avenue to pursue. Note that I can still define my tables using the "declarative" idioms--the underlying tables can be accessed via the `__table__` attribute. 

_On the other hand_ (and the reason I'm not just starting off writing the backend using Core), 12s for 100k rows doesn't actually sound that bad--if I could parse a million logs in a couple of minutes, that would be an _enormous_ improvement on what I have now. Basically what I'm saying is that even doing things the slow ORM way, there's a good chance this won't be a significant bottleneck.

Bottom line: this is something to keep track of, but 50/50 this gets closed as "won't fix"

**Comment by Antar1011** [comment]
Working on this on [`sa-core` branch](https://github.com/Antar1011/Onix/tree/sa-core)

**Comment by Antar1011** [comment]
Resolved by #55
